### DISCUSSION TITLE
Container Queries

### DISCUSSION THREAD

**Discussion by nicklassandell** [unocss/unocss#3543] [General]
Are [Container Queries ](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_container_queries)supported? Tailwind has a plugin for it which is working great: https://github.com/tailwindlabs/tailwindcss-container-queries

I tried using the same syntax in Uno but no luck. Is it implemented under a different syntax, or is it not implemented yet?

It is available in all evergreen browsers: https://caniuse.com/css-container-queries

Thanks

**Comment by nicklassandell** [comment]
Appears to be implemented but afaik you can't define the container (like in tailwind with \@container) or define the width query inline, you can only use the predefined container widths (sm/md/lg etc) from your uno config.

https://github.com/unocss/unocss/blob/main/packages/preset-mini/src/_variants/container.ts#L13

**Comment by mortona42** [comment]
I am having an issue where container queries are compiled out of order. They have min-width values, so if the smaller one is after the larger one, it takes over at the wrong breakpoint.

This was working with 59.3 before I updated to 61.0.
